backend/capellacollab/projects/capellamodels/models.py

```diff
@@ -31,39 +38,96 @@ class CapellaModelType(enum.Enum):
     LIBRARY = "library"
 
 
-class DB_CapellaModel(Base):
-    __tablename__ = "capella_models"
+class NewModel(BaseModel):
+    name: str
+    description: t.Optional[str]
+    tool_id: int
+
+
+class EmptyModel(BaseModel):
+    name: str
+    description: t.Optional[str]
+    tool_id: int
+    version_id: int
+    type_id: int
 
-    id = Column(Integer, primary_key=True, index=True)
-    name = Column(String, index=True)
+
+class ToolDetails(BaseModel):
+    version_id: int
+    type_id: int
+
+
+class Model(Base):
```

Comment:
Usually, we name the database classes with the `Database` prefix to avoid conflicts. Therefore, the class should be named `DatabaseModel` or even better `DatabaseCapellaModel`.